recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
-{% set version = "1.4.0" %}
-{% set commit = "b62c22e52d37fa9580d85c5a2a2d73bd8e268364" %}
-{% set sha256sum = "2fa5704fc1164f68508e3050d30a6436dd56434e1a592d8ddf4afd68f62a7c01" %}
+{% set version = "1.4.0.post0" %}
```

Comment:
@wesm `1.4.1`, `1.5.0` or `1.4.0.post0`?

Reply:
I recommend 1.5.0pre to avoid conflicting with 0.9.0 installs

Reply:
`post`, since there has not been a parquet-cpp release since `1.4.0`.

Reply:
> Wes has responded later but Phillip's comment is at the bottom (might be a github bug?)
I guess this should be `1.5.0pre` then.

Reply:
LGTM